definitions/private-law/json/ComplexTypes/UploadAdditionalApplications/2_C2DocumentBundle/ComplexTypes.json

```diff
@@ -11,167 +11,163 @@
     "Searchable": "N"
   },
   {
     "LiveFrom": "20/11/2022",
     "ID": "C2DocumentBundle",
-    "ListElementCode": "type",
-    "FieldType": "FixedRadioList",
-    "FieldTypeParameter": "C2ApplicationTypeEnum",
-    "ElementLabel": "Application type",
+    "ListElementCode": "caReasonsForC2Application",
+    "ElementLabel": "Reason for application",
+    "HintText": "Select all that apply",
+    "FieldType": "MultiSelectList",
+    "FieldTypeParameter": "C2AdditionalOrdersRequestedCa",
     "SecurityClassification": "Public",
     "Searchable": "N"
   },
   {
     "LiveFrom": "20/11/2022",
     "ID": "C2DocumentBundle",
-    "ListElementCode": "uploadedDateTime",
-    "FieldType": "Text",
-    "ElementLabel": "Date and time of upload",
+    "ListElementCode": "daReasonsForC2Application",
+    "ElementLabel": "Reason for application",
+    "HintText": "Select all that apply",
+    "FieldType": "MultiSelectList",
+    "FieldTypeParameter": "C2AdditionalOrdersRequestedDa",
     "SecurityClassification": "Public",
     "Searchable": "N"
   },
   {
     "LiveFrom": "20/11/2022",
     "ID": "C2DocumentBundle",
-    "ListElementCode": "author",
-    "FieldType": "Text",
-    "ElementLabel": "Uploaded by",
+    "ListElementCode": "combinedReasonsForC2Application",
+    "ElementLabel": "Reason for application",
+    "HintText": "Select all that apply",
+    "FieldType": "MultiSelectList",
+    "FieldTypeParameter": "CombinedC2AdditionalOrdersRequested",
     "SecurityClassification": "Public",
     "Searchable": "N"
   },
   {
     "LiveFrom": "20/11/2022",
     "ID": "C2DocumentBundle",
-    "ListElementCode": "othersNotified",
-    "Comment": "Deprecated",
+    "ListElementCode": "otherReasonsFoC2Application",
     "FieldType": "Text",
-    "ElementLabel": "People notified",
+    "ElementLabel": "Other reasons for C2 Application",
     "SecurityClassification": "Public",
-    "Searchable": "N"
+    "Searchable": "N",
+    "FieldShowCondition": "applicantName=\"DO_NOT_SHOW\""
   },
   {
     "LiveFrom": "20/11/2022",
     "ID": "C2DocumentBundle",
-    "ListElementCode": "description",
-    "FieldType": "TextArea",
-    "ElementLabel": "Notes",
+    "ListElementCode": "hearingList",
+    "ElementLabel": "Hearing List",
+    "FieldType": "DynamicList",
     "SecurityClassification": "Public",
     "Searchable": "N"
   },
   {
     "LiveFrom": "20/11/2022",
     "ID": "C2DocumentBundle",
-    "ListElementCode": "nameOfRepresentative",
+    "ListElementCode": "requestedHearingToAdjourn",
+    "ElementLabel": "Hearing to adjourn",
     "FieldType": "Text",
-    "ElementLabel": "Name of representative",
     "SecurityClassification": "Public",
     "Searchable": "N"
   },
   {
     "LiveFrom": "20/11/2022",
     "ID": "C2DocumentBundle",
-    "ListElementCode": "usePbaPayment",
-    "FieldType": "YesOrNo",
-    "ElementLabel": "Paid with PBA",
+    "ListElementCode": "uploadedDateTime",
+    "FieldType": "Text",
+    "ElementLabel": "Date submitted",
```

Comment:
if fieldtype change then definitely ES exception will be thrown during ccd deployment. Please check

Reply:
noted. will run a test in sublive.